packages/packages/vue/@tryabby/vue/src/useAbby.ts

```diff
@@ -0,0 +1,13 @@
+// packages/vue/@tryabby/vue/src/useAbby.ts
+import { ref, onMounted } from 'vue';
+import { getVariant, onAct } from '@tryabby/core';
```

Comment:
this is **not** how the Abby core works. The core exports a class that needs to be initialized and then you can call those functions. Please have a look at the react implementation to see how this is done